exercises/isogram/README.md

```diff
@@ -42,7 +42,8 @@ The [exercism/rust](https://github.com/exercism/rust) repository on GitHub is th
 If you want to know more about Exercism, take a look at the [contribution guide](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/README.md).
 
 [help-page]: http://exercism.io/languages/rust
-[crates-and-modules]: http://doc.rust-lang.org/stable/book/crates-and-modules.html
+[modules]: https://doc.rust-lang.org/book/second-edition/ch07-00-modules.html
+[cargo]: https://doc.rust-lang.org/book/second-edition/ch14-00-more-about-cargo.html
```

Comment:
This footnote isn't referenced at all, and is [stripped out of the rendered document](https://github.com/petertseng/exercism-rust/blob/ae6f16d2d09434816b027dd5051b086cc4114863/exercises/isogram/README.md).